llvm/include/llvm/SYCLLowerIR/ComputeModuleProperties.h

```diff
@@ -0,0 +1,45 @@
+//===- ComputeModuleProperties.h - compute properties for module -===//
```

Comment:
```suggestion
//===- ComputeModuleProperties.h - compute properties for module ----------===//
```
Usually, this part is exactly 80 characters long.

Reply:
fixed in latest commit, thx